docs/site/pages/plugins/asset-provider.mdx

```diff
@@ -9,7 +9,61 @@ The Asset Provider Plugin enables users to easily register UI components to rend
 
 ### Usage
 
+Use the Asset Provider plugin the register your custom assets. In this example, we have components that are rendering a customer asset from type `custom-asset` and `custom`.
```

Comment:
We probably want to show an example of actually using the plugin here to load React components into React Player